### ISSUE TITLE
cannot import name 'decode' from 'PyInstaller.utils.win32.versioninfo' using pyinstaller==5.8.0

### ISSUE THREAD

**Issue by kenbunji** [flet-dev/flet#1060]
Flet fails to execute pack under below condition.

* flet==0.3.2
  flet==0.4.0 has same bug
* pyinstaller==5.8.0
* Windows OS

**Description**

How to raise the error:
  * just create pack
  ```
  > flet pack some_project.py
  ```

Reason:
  * pyinstaller 5.8.0 deprecated decode function from PyInstaller/utils/win32/versioninfo.py
  * see below commit
    https://github.com/pyinstaller/pyinstaller/commit/f57e15ae14d2370cba7a14cfae97d2c29b5c8154

Validation code to suppress the error:
flet\__pyinstaller\win_utils.py
```Python: flet\__pyinstaller\win_utils.py
# from PyInstaller.utils.win32.versioninfo import decode
from PyInstaller.utils.win32.versioninfo import read_version_info_from_executable as decode
```
or use pyinstaller==5.7.0 (it will work)

**Additional information you deem important (e.g. issue happens only occasionally):**

**Flet version**:
flet==0.3.2

**PyInstaller version**:
pyinstaller==5.8.0

**Operating system**:
Windows 11

**Comment by iron3oxide** [comment]
duplicate of #1031, should be fixed in 0.4.1 and prerelease versions

**Comment by FeodorFitsner** [comment]
Yep, it's been fixed in just released Flet 0.4.1. Please update to the latest version.
